recipes/gecos/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set version = "1.2.0" %}
+{% set number = "0" %}
+
+package:
+  name: gecos
+  version: {{ version }}
+
+source:
+  url: https://github.com/biotite-dev/gecos/archive/v{{ version }}.tar.gz
+  sha256: 48fc10ae4d018531b21815bedb49b095b33e13394ec3a69266d28e7c7c4bc01e
+
+build:
+  number: {{ number }}
```

Just avoid templating it and put it here directly:
```suggestion
  number: 0
```